src/Evolution/Systems/GeneralizedHarmonic/Constraints.hpp

```diff
@@ -144,4 +144,40 @@ void two_index_constraint(
     const tnsr::iaa<DataType, SpatialDim, Frame>&
         three_index_constraint) noexcept;
 // @}
+
+// @{
+/*!
+ * \brief Computes the generalized-harmonic 4-index constraint.
+ *
+ * \details Computes the independent components of the generalized-harmonic
+ * 4-index constraint. The constraint itself is given by Eq. (45) of
+ * \cite Lindblom:2005qh,
+ * \f{eqnarray}{
+ * C_{ijab} = 2 \partial_{[i}\Phi_{j]ab},
+ * \f}
+ * where \f$\Phi_{iab} = \partial_i\psi_{ab}\f$. Because the constraint is
+ * antisymmetric on the two spatial indices, here we compute and store
+ * only the independent components of \f$C_{ijab}\f$. Specifically, we
+ * compute
+ * \f{eqnarray}{
+ * D_{iab} \equiv \frac{1}{2} \epsilon_{i}{}^{jk} C_{jkab}
+ * = \epsilon_{i}{}^{jk} \partial_j \Phi_{kab},
```

Comment:
I think the empty braces `{}` are not actually doing anything.

Reply:
You are right. Somehow in the online latex renderer I used, this was not the case, but rendering locally with latex renders the offset.

I think `{\epsilon_{i}}^{jk}` might be more robust.

Reply:
The version as coded seems robust enough to me...renders correctly when I build the docs and open them in my browser. So I'd like to leave this as-is.